### PR TITLE
Correct order of TABLE removal

### PR DESCRIPTION
**Pull request by borisroman**

### CODE REVIEW

**Comment by borisroman** [comment]
Ping @wilderrodrigues

**Comment by wilderrodrigues** [comment]
Now yes! :)

:+1: 

```
[INFO] 
[INFO] --- maven-install-plugin:2.3.1:install (default-install) @ cloud-developer ---
[INFO] Installing /data/git/cs2/cosmic/cosmic-core/developer/pom.xml to /root/.m2/repository/cloud/cosmic/cloud-developer/5.0.0-SNAPSHOT/cloud-developer-5.0.0-SNAPSHOT.pom
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 16.101s
[INFO] Finished at: Tue Jan 26 14:49:06 GMT 2016
[INFO] Final Memory: 38M/235M
[INFO] ------------------------------------------------------------------------
[root@cs2 cosmic-core]#
```

File: setup/db/db/schema-480to500-cleanup.sql [hunk 1]
```diff
@@ -3,10 +3,10 @@
...
 
 # Remove NetApp plugin
+DROP TABLE IF EXISTS `cloud`.`netapp_lun`;
 DROP TABLE IF EXISTS `cloud`.`netapp_volume`;
 DROP TABLE IF EXISTS `cloud`.`netapp_pool`;
-DROP TABLE IF EXISTS `cloud`.`netapp_lun`;
 
 # Remove BigSwitch plugin
+DROP TABLE IF EXISTS `cloud`.`external_bigswitch_vns_devices`;
 DROP TABLE IF EXISTS `cloud`.`external_bigswitch_bcf_devices`;
```
Comment (remibergsma) [inline setup/db/db/schema-480to500-cleanup.sql]: @borisroman FYI: There is also a table called 'cloud.brocade_network_vlan_map', at least in NL2.

Comment (borisroman) [inline setup/db/db/schema-480to500-cleanup.sql]: Thanks! Will add them to the drop list.

**Pull request merged**